### PR TITLE
[use-import-remote] exporting useImportRemote hook

### PR DESCRIPTION
**Pull request by dleroux**
## Description

This exports the `useImportRemote` hook wasn't exported from the package. 

## Type of change

- [x]  Patch: Bug (non-breaking change which fixes an issue)
- [ ]  Minor: New feature (non-breaking change which adds functionality)
- [ ]  Major: Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] I have added a changelog entry, prefixed by the type of change noted above (Documentation fix and Test update does not need a changelog as we do not publish new version)

### CODE REVIEW

**Comment by dleroux** [comment]
Not sure if this should be considered a bug or a patch?

File: packages/react-import-remote/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
 
+- Exported `useImportRemote` hook and `Status`([1634](https://github.com/Shopify/quilt/pull/1634))
```
Comment (dleroux) [inline packages/react-import-remote/CHANGELOG.md:10]: Everything has been released not sure if this is the proper way to add an unreleased change?

**Review by cartogram — APPROVED** [review]

File: packages/react-import-remote/CHANGELOG.md (hunk 1 above)
Comment (cartogram) [reply packages/react-import-remote/CHANGELOG.md:10]: looks right 👍

**Comment by cartogram** [comment]
@dleroux I think patch

**Pull request merged**